docs/get-started-with-datahub.md

```diff
@@ -0,0 +1,62 @@
+# Get Started With DataHub
```

Comment:
The main purpose of this document is to provide the "Get Started" section with a landing page of sorts, so it's intentionally sparse.